revscoring/features/wikitext/datasources/tokenized.py

```diff
@@ -0,0 +1,437 @@
+import re
+
+from deltas import wikitext_split
+
+from ....datasources import Datasource
+from ....datasources.meta import filters, frequencies, mappers
+
+
+class Revision:
```

Comment:
As discussed in chat, this is only necessary python 2.x.  In python 3, inheritance from `object` is implied.